.travis.yml

```diff
@@ -18,17 +23,22 @@ install:
     conda config --add channels conda-forge --force
     conda config --set channel_priority strict
     conda config --set safety_checks disabled
+    conda create --name TEST python=$PY --file requirements.txt --file requirements_dev.txt --file requirements_docs.txt --file requirements_plus.txt
```

Comment:
This is the "core" of this PR, it will create the environment for each python in the matrix based on the same files used in `setup.py`, so the conda test env and a pip installation are installing the same packages.